logbook-core/src/main/java/org/zalando/logbook/BodyFilters.java

```diff
@@ -10,15 +10,15 @@
     }
 
     public static BodyFilter defaultValue() {
-        return accessToken();
+        return replaceProperty("access_token|open_id|id_token", "XXX");
```

Comment:
A couple of things I noticed:
- having multiple properties separated by a pipe leaks the implementation detail that we're using a regular expression
- I'd propose to a parameter of type `Set<String>`
- property names are not escaped/quoted, i.e. it allows for "regex injection", see [`Pattern.quote(String)`](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#quote-java.lang.String-)
- so basically: `properties.stream().map(Pattern::quote).collect(joining("|"))`
- the whole idea only works for string properties, which makes it immediately less useful
- I'd rename it `replaceStringProperties`